utils/_context/_scenarios/endtoend.py

```diff
@@ -182,7 +182,7 @@ def __init__(
         agent_env=None,
         tracer_sampling_rate=None,
         appsec_enabled=True,
-        iast_enabled=True,
+        iast_enabled=False,
```

Comment:
You'll need to set `iast_enabled=True` for scenario that test it (no a lot of them though)